frontend/src/components/SignUp.js

```diff
@@ -173,13 +190,20 @@ export class SignUp extends React.Component {
                                 onChange={this.handleInputChange}
                             />
                         </div>
-                        <input type='hidden' name='success_uri' value='/account/signup/success' />
-                        <input
-                            type='hidden'
-                            name='error_uri'
-                            value={this.props.location.pathname}
+                        <div style={{ display: this.state.captchaPassed && 'none' }}>
+                            Please complete the captcha below.
```

Comment:
I think we probably don't need this text

Reply:
Sure, this has been removed since the reCAPTCHA should be self-explanatory.